app/components/form/fields/DisksTableField.tsx

```diff
@@ -99,7 +110,7 @@ export function DisksTableField({ control }: { control: Control<InstanceCreateIn
           onDismiss={() => setShowDiskCreate(false)}
         />
       )}
-      {showDiskAttach && (
+      {showDiskAttach && !disabled && (
```

Comment:
This means that if you have the modal open and click submit, it will close, but if there's an error and we land back on the form (however we do that) it will pop back open?

Reply:
I believe that is what should happen, as it isn't affecting the showDiskAttach (or Create) values. Will try testing that scenario to verify.

Reply:
Turns out you can't even submit the form with the modals open, as they lose focus and disappear. So I've removed the `!disabled` from those two spots, as they're moot.